_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: Release 1.0.0, 1.0.1, 1.1.1
+---
+
+A new release of MDAnalysis now available!
+Or rather, three releases have been made for the 1.x version of MDAnalysis, so we just talk about all of them in a single post.
+
+
+This version brings a multitude of **fixes**, [**deprecations**](#deprecations), and **new features**.
+Some highlights are given below, whilst the release notes(someone make this a link to the right place please) list all the changes in this version.
+
+This release completes, albeit with a small delay, the first phase of our [roadmap][]. It is the last version which will support python 2.7 and 3.5. Our next release, 2.0.0, which will be made available soon, will include major API changes and will support python 3.6+
+
+## GSoC? 
+
+## Notable new additions
+
```

Comment:
Stable API (for 1.x). Semantic versioning guarantees no backward incompatible changes within 1.x

Reply:
I mentioned this above, does that suffice?

Reply:
yes